styles/style.css

```diff
@@ -0,0 +1,237 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500;600&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-size: 30px;
+    font-family: 'Montserrat', sans-serif;
+    color: black;  
+}
+
+
+.page {
+    position: relative;
+    width: 100%;
+    height: 100%;
+    background-image: url(img/QuizBackground.png);
+    background-size: cover;
+    padding: 3%;
+}
+
+.menu{
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    width: 100%;
+    margin-bottom: 2%;
+}
+
+.menu__logo {
+    width: 25%;
+}
+
+.menu__list {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    width: 65%;
+    height: 110px;
+    background-color: white;
+    border-radius: 16px;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25), 4px 4px 40px rgba(255, 0, 0, 0.9);
+}
+
+.menu__item {
+    position: relative;
+    padding: 10px;
+    width: 20%;
+    text-decoration: none;
+    font-weight: 600;
+    text-align: center;
+    color: black;
+    
```

Comment:
tu jakieś zbędne enterki

Reply:
trzeba dać na całym autoformatowanie to i entery znikną :)

Reply:
niby mam włączone beautify on save ale jakoś widocznie nie działa :( ja tych enterów u siebie w vsc nie widzę. Jakieś rady?

Reply:
formatuj ręcznie przed commitem :) jak chcesz potestować ten format on save to usuń spacje po klasie typu .costam{ ...}, zapisz plik i zobacz czy poprawiło

Reply:
sformatowałabym ręcznie, problem, że ja w vsc enterów nie widzę... dla mnie każda linijka jest jedna pod drugą.

Reply:
zrobiłam też tak jak napisałeś, ale ja żadnej różnicy niestety nie widzę.